build.gradle

```diff
@@ -70,6 +70,8 @@ configure(opentelemetryProjects) {
         it.options.errorprone.disable("UnnecessaryAnonymousClass")
         // "-Xep:UnnecessaryAnonymousClass:OFF"
 
+        it.options.errorprone.disable("UnusedNestedClass")
```

Comment:
what is the use-case for an unused nested class that we wouldn't want to warn about it?

Reply:
Damn... My benchmark class has a `ThreadState` inner class that error prone disliked. I will take a look if I can remove this.

Reply:
Removed this line